testing/fixtures/fixtures.go

```diff
@@ -15,7 +15,7 @@ type Project struct {
 	Name   string
 	URL    string
 	Commit string
-	Env    map[string]string
+	// Env    map[string]string
```

Comment:
Just delete this if you're going to remove it.

(Which you shouldn't, as I've mentioned above.)